run/README.namelist

```diff
@@ -691,6 +691,20 @@ Namelist variables for controlling the adaptive time step option:
  num_urban_nf                        = 1! (= 10 if BEM active)        number of grid levels in the floors (nf_u in BEM header)
  num_urban_nbui                      = 1! (= 15 if BEM active)        maximum number of types of buildings in an urban class (nbui_max in BEM header)
 
+ sf_surf_irr_scheme (max_dom)        = 0: no irrigation scheme
+                                     = 1: channel scheme (in Noah only)
+                                     = 2: drip scheme
+                                     = 3: sprinkler scheme
+ 
+ irr_daily_amount (max_dom)          = daily equivalent irrigation water amount in mm (e.g. 5.7)
+ irr_start_hour (max_dom)            = UTC start hour for irrigation (e.g. 5)
```

Comment:
@arjanna 
Arianna,
For the "hour" options, please indicate if these are "real" or "integer" values.